src/draw/DefaultLayout.js

```diff
@@ -20,9 +20,12 @@ class DefaultLayout {
   /**
    * Does nothing to components.
    * One can override this method to implement an algorithm for automatic components layout.
+   * @param {string} [containerId] - Container of which we must arrange the children. If not given,
```

Comment:
`The container within which we need to organize the children, and if not specified, all components will be reorganized.` (for all)

Reply:
done

Reply:
done